annet/cli.py

```diff
@@ -120,6 +124,62 @@ def _show_device_dump_items(devices):
         )
 
 
+@subcommand(cli_args.ShowGeneratorsOptions, parent=show)
+def show_generators(args: cli_args.ShowGeneratorsOptions):
+    """ List applicable generators (for a device if query is set) """
+    arg_gens = cli_args.GenOptions(args)
+    with get_loader(arg_gens, args) as loader:
+        device: Device | None = None
+        devices = loader.devices
+        if len(devices) == 1:
+            device = devices[0]
+        elif len(devices) > 1:
+            get_logger().error("show-generators support only one device")
```

Comment:
supports a single device

Reply:
Reworded to `cannot show generators for more than one device at once`